cli/src/pcluster/cli.py

```diff
@@ -82,6 +82,11 @@ def stop(args):
     pcluster_stop.stop(args)
 
 
+def build_image(args):
+    pcluster.build_image(args)
+
+
+# moved
```

Comment:
Feel free to delete this code if useless

Reply:
Done